taxcalc/tests/test_utils.py

```diff
@@ -226,7 +226,8 @@ def test_weighted_share_of_total():
     df = DataFrame(data=data, columns=['tax_diff', 's006', 'label'])
     grped = df.groupby('label')
     diffs = grped.apply(weighted_share_of_total, 'tax_diff', 42.0)
-    exp = Series(data=[16.0 / 42., 26.0 / 42.0], index=['a', 'b'])
+    exp = Series(data=[16.0 / (42. + 1e-3), 26.0 / (42.0 + 1e-3)],
```

Comment:
If you added code like this:

```
from taxcalc.utils import EPSILON
```

and then did:

```
exp = Series(data=[16.0 / (42. + EPSILON), 26.0 / (42.0 + EPSILON)],
```

then you would never have to change this if `EPSILON` changed.